pkg/istio/virtualservice_test.go

```diff
@@ -17,14 +17,20 @@ var _ = Describe("Operations for istio VirtualService kind", func() {
 		err            error
 		virtualService istionetwork.VirtualService
 		yaml           string
+		ref            *model.Ref
 	)
 
 	Context("mutators", func() {
 
 		JustBeforeEach(func() {
 			err = k8yaml.Unmarshal([]byte(yaml), &virtualService)
 		})
-
+		BeforeEach(func() {
+			ref = &model.Ref{
```

Comment:
`ref` -> `detailsDeploymentRef`?

Reply:
A large section of this test case is being redone in an upcoming PR. Leave it for now.